### PR TITLE
Danger changelog rule

### PR DESCRIPTION
**Pull request by piv199**
## Checklist

- [x] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [x] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/DateExtensions.swift [hunk 1]
```diff
@@ -33,7 +33,7 @@ extension Date {
             self = asctime
             return
         }
-        //self.init()
+        ///self.init()
```
Comment (Khalian) [inline Sources/DateExtensions.swift]: Can we revert this change?

File: Dangerfile [hunk 2]
```diff
@@ -1,3 +1,12 @@
+#Ensure changelog entry
+unless github.pr_body.include? "- [x] Trivial change (doesn't require changelog)" 
+	if (git.modified_files.grep(/^Sources\//).count > 0) || (github.pr_body.include? "- [x] New Extension")
+		if git.modified_files.grep(/^CHANGELOG.md/).count == 0 
+			fail("Making non-trivial change requires changelog entry! Please, set trivial change or add entry to changelog.")
+		end
+	end
+end
+
```
Comment (Khalian) [inline Dangerfile]: We can squash the commits into one.

Comment (piv199) [inline Dangerfile]: @Khalian easy to say 🎱 I've done it with Source tree! Yeah! Cheers! 👍

**Comment by piv199** [comment]
should be merged as soon as #266 is merged
